canvas_account_admin_tools/settings/base.py

```diff
@@ -74,6 +74,7 @@
     'icommons_common',
     'icommons_ui',
     'lti_permissions',
```

Comment:
probably want to remove `lti_permissions` here, I think?

Reply:
^ Noted during standup that `lti_permissions` is needed initially for the migrations in `lti_school_permissions` to run. This can be removed after release.